addons/account/wizard/account_payment_register.py

```diff
@@ -578,21 +576,23 @@ def _convert_to_wizard_currency(self, installments):
             else:
                 # Foreign currency on payment different than the one set on the journal entries.
                 total_amount += comp_curr._convert(amount_residual, wizard_curr, self.company_id, self.payment_date)
-        return abs(total_amount)
+        return total_amount
```

Comment:
mmh? you sure it was there for a reason... from a philosophical point of view: registering a payment for a customer invoice or for a customer credit note, is always registered as a positive amount: it's just the money flow that is inversed, and I felt it logical that way :eyes:

Reply:
edit: ho mmh maybe you kept that but it's made later on.... :thinking: do you confirm?

Reply:
Yes, the only use of this method have an `abs` afterwards.
I had to remove it here due to the fact that now we can mix installments with epd.
It's not doable on a single move (as it's prevented on the payment term), but now you can associate a move with installments and a move with epd.
If you associate credit note and bills when you pay, it caused issues in `full_amount_for_difference`, if you do the abs before the sum (revealed in the test here https://github.com/odoo/odoo/pull/180662/files#diff-dfdcb8828d112aa28986bcaab7e6d29f7ce1aa3b6a5556d92ea80d4169090eb9R1509 )